### PR TITLE
Adds action buttons for slimes

### PR DESCRIPTION
**Pull request by Incoming5643**
What it says on the tin, action buttons for feeding, evolving, and splitting.

Slime tab is still there, but no one ever really needs to use it now.

### CODE REVIEW

**Comment by Xhuis** [comment]
thumbsup

File: tgstation.dme [hunk 1]
```diff
@@ -255,7 +255,6 @@
 #include "code\datums\spells\mind_transfer.dm"
 #include "code\datums\spells\projectile.dm"
 #include "code\datums\spells\santa.dm"
-#include "code\datums\spells\shapeshift.dm"
```
Comment (Xhuis) [inline tgstation.dme]: I think you did the thing

**Comment by Incoming5643** [comment]
Also a gentle reminder that you can also start feeding as a slime by dragging and dropping yourself on whatever you want to eat. It's a very useful shortcut.

**Comment by KorPhaeron** [comment]
Neat

**Comment by Aranclanos** [comment]
the CheckRemoval() and IsAvailable() from both buttons are a copypaste from eachother
also the amount_grown needs to be a define

**Comment by Incoming5643** [comment]
Are you really suggesting I subtype two buttons from an already subtyped set of three buttons to save six lines?

Also CheckRemoval isn't the same between the two, one checks for adult slime status, and the other checks for the lack of it.

**Comment by duncathan** [comment]
@Incoming5643 fix aran's concerns

**Comment by Incoming5643** [comment]
fffffffine

@duncathan can you merge this sooner rather than later, everyone and there mom loves touching actions.dmi

**Pull request merged**